com.github.parnoldx.nasc.json

```diff
@@ -0,0 +1,165 @@
+{
+  "app-id": "com.github.parnoldx.nasc",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "sdk": "org.gnome.Sdk",
+  "command": "nasc",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--talk-name=org.gtk.vfs",
+    "--talk-name=org.gtk.vfs.*",
+    "--filesystem=xdg-desktop",
+    "--filesystem=xdg-documents",
+    "--filesystem=xdg-public-share",
+    "--filesystem=xdg-run/dconf",
+    "--filesystem=~/.local/share/nasc/",
```

Comment:
`$XDG_DATA_HOME` is set to another private location for you automatically so you shouldn't need to pass this through.

Reply:
Oh it does hardcode this path: https://github.com/parnold-x/nasc/blob/89c3d753fd0a82a82e2cddac7a3227bfcb979966/src/NascSettings.vala#L27

That should be patched out to respect the correct xdg directory: https://valadoc.org/glib-2.0/GLib.Environment.get_user_data_dir.html